transitions/registration.js

```diff
@@ -313,17 +313,15 @@ module.exports = {
             patientId = doc.patient_id,
             patientNameField = _.first(options.params) || 'patient_name';
 
-        utils.getRegistrations({
-            db: db,
-            id: patientId
-        }, function(err, registrations) {
-            if (err) {
+        db.medic.get('patient-' + patientId, function(err, patientContact) {
```

Comment:
This won't work with any 2.9.x data or anyone migrated from 0.4.x. To rely on this ID format we'd need to write a migration to update patient uuids (which is a pain), but otherwise the ID format is pretty much useless. The alternative is to query the view.